packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-linking-step/wallet-linking-step.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Text, ThemedScrollbars } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletLinkWrapper, { TWalletLinkWrapper } from '../wallet-link/wallet-link-wrapper';
+import './wallet-linking-step.scss';
+
+type TWalletLinkingStep = {
+    data: { title: string; wallets: TWalletLinkWrapper[] };
+};
+
+const WalletLinkingStep = observer(({ data }: TWalletLinkingStep) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    return (
+        <div className='wallet-linking-step'>
+            <Text as='div' color='prominent' size={is_mobile ? 's' : 'm'} weight='bold'>
+                {data.title}
+            </Text>
+            <Text as='div' className='wallet-linking-step__description' color='prominent' size={is_mobile ? 'xs' : 's'}>
+                {localize('This is how we link your accounts with your new Wallet.')}
```

Comment:
Please use `Localize` component here whenever we using the Text component instead of localize 👍